src/components/LogOut.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+import { getAuth, signOut } from 'firebase/auth';
+import { onMounted } from 'vue';
+
+import { REDIRECT_URI_PROP } from '../common';
+import { router } from '../router';
+
+const auth = getAuth();
+
+const props = defineProps(REDIRECT_URI_PROP);
+
+function logOut() {
```

Comment:
Should we also clear the authentication cookies or will the Apigee logic be added in a follow up CL?

Reply:
Apigee logic will be added in a follow up CL. No cookies are currently set.